test/fail/ok/issue-4323.tc.ok

```diff
@@ -1,4 +1,4 @@
-issue-4323.mo:4.1-6.2: type error [M0096], expression of type
+issue-4323.mo:4.1-6.2: type error [M0192], body of type
```

Comment:
Currently the source range extends to the whole thing, would you insist on restricting to the body only?

Reply:
Probably doesn't matter match, tbh

Reply:
This is like `M0134` treats it as well.